types.go

```diff
@@ -165,7 +166,7 @@ func (r *request) URL(serverURL string) (string, error) {
 		r.service, // service
 		version,   // version
 		r.profile, // profile
-		"polyline(" + r.geoPath.Polyline() + ")", // coordinates
+		"polyline(" + url.QueryEscape(r.geoPath.Polyline()) + ")", // coordinates
```

Comment:
Should it be https://golang.org/pkg/net/url/#PathEscape ? I'm not sure what's the difference exactly, but seems like polyline is a part of path in resulting URL - not a part of query

Reply:
done